frontend/elements/src/i18n/fr.ts

```diff
@@ -0,0 +1,124 @@
+import { Translation } from "./translations";
+
+export const fr: Translation = {
+  headlines: {
+    error: "Une erreur s'est produite",
+    loginEmail: "Se connecter ou s'inscrire",
+    loginFinished: "Connexion réussie",
+    loginPasscode: "Entrez le code d'accès",
+    loginPassword: "Entrez le mot de passe",
+    registerAuthenticator: "Enregistrer une passkey",
```

Comment:
"Enregistrer une clé d'accès"

We could use the french term for passkey "Clé d'accès". Though the english term "passkey" IS used in French language, many French article mention it. And it is shorter.